open-data-fabric.md

```diff
@@ -700,7 +700,7 @@ date_of_birth DATE,
 Supported types:
 
 |    DDL Type    |                                        Parquet Type                                        |
-| :------------: | :----------------------------------------------------------------------------------------: |
+|:--------------:|:------------------------------------------------------------------------------------------:|
```

Comment:
This is how VSCode markdown extension is auto-formatting these tables. Unless we can somehow configure this for everyone I'd rather revert for now as this will cause me a lot of headache.

In future we could push auto-formatting into some dockerized command, so everyone uses the same tool.